Common/UnitDefinitions/Power.json

```diff
@@ -12,6 +12,11 @@
     {
       "SingularName": "Watt",
       "PluralName": "Watts",
+      "BaseUnits": {
+        "L": "Meter",
+        "M": "Kilogram",
+        "T": "Second"
+      },
```

Comment:
Power had not defined base units for any units, so I added this to pass one of the test cases on decimal quantities.